### PR TITLE
[#91629492] add environment variable to force HTTPS

### PR DESCRIPTION
**Pull request by afeld**
* [ ] Set `FORCE_HTTPS=true` on production

### CODE REVIEW

**Comment by afeld** [comment]
Updated to use [a middleware](https://github.com/tobmatth/rack-ssl-enforcer) (which does stuff like force secure cookies).

/cc @konklone

**Comment by konklone** [comment]
:+1: :us:

File: .env.example [hunk 1]
```diff
@@ -15,6 +15,7 @@ MYUSA_SECRET: consumer_secret_key
 # DEFAULT_URL_PORT
 # DEFAULT_URL_SCHEME
 # FLAG_ITEM_COMMENT_UI
+# FORCE_HTTPS=true
```
Comment (cmc333333) [inline .env.example]: Just checking - will this need to be overridden on dev boxes?

Comment (afeld) [inline .env.example]: You see that it's commented out? HTTPS-only is off by default.

Comment (cmc333333) [inline .env.example]: All the lines in `app/assets/javascripts/application.js` are commented, too, but things aren't always what they appear in rails land ;)

Comment (afeld) [inline .env.example]: Lol, touché.

**Pull request merged**